[ayucloud.services]
email  lgmcxpgan606@grr.la
last_checkin  2025-06-13 10:19:01.416685+08:00
node_n  99
sub_info  0B  0B  2025-06-13 14:44:17+08:00  (剩余 0B -13 days, 21:08:57.152942)
sub_url  https://network.ayucloud.nl/link/0Ziuj9eTXdQtsjMm?clash=1
time  2025-06-12 14:44:19.060259+08:00
保存base64订阅失败  no content
保存clash订阅失败  'NoneType' object is not iterable
尝试签到失败  ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
更新订阅链接/续费续签失败  注册失败(qbcynhqtd531@gmail.com): HTTPSConnectionPool(host='ayucloud.services', port=443): Max retries exceeded with url: /user/register (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f44a5379900>, 'Connection to ayucloud.services timed out. (connect timeout=5)'))
